pkgs/tools/misc/starship/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgs }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "starship";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "starship";
+    repo = "starship";
+    rev = "v${version}";
+    sha256 = "045lq4nsrdssmqbcj0551f2c5qd2rcvhs8gr4p4iniv7s89yz1xl";
+  };
+
+  buildInputs = with pkgs; [ openssl pkgconfig ];
+
+  cargoSha256 = "126y8q19qr37wrj6x4hqh0v7nqr9yfrycwqfgdlaw6i33gb0qam9";
+
+  meta = with stdenv.lib; {
+    description = "The cross-shell prompt for astronauts";
```

Comment:
How about (from the first sentence of the guide) "Starship is the minimal, blazing fast, and extremely customizable prompt for any shell"? It's a little less... cryptic.

Reply:
You are totally right. Thanks